src/ontology/mondo-edit.obo

```diff
@@ -89545,8 +89545,10 @@ is_a: MONDO:0000594 {source="DOID:0060041", source="NCIT:C88412"} ! pervasive de
 id: MONDO:0005259
 name: Asperger syndrome
 def: "A disorder most often diagnosed in the pediatric years in which the individual displays marked impairment in social interaction and a repetitive, stereotyped pattern of behavior. The individual, however, displays no delay in language or cognitive development, which differentiates Asperger Syndrome from autism." [NCIT:C97159]
+comment: The name Aspergers is controversial, however, it seems to be the community accepted name. {xref="https://github.com/monarch-initiative/mondo/issues/705", xref="https://blogs.scientificamerican.com/observations/the-problem-with-aspergers/", xref="https://github.com/EBISPOT/efo/issues/275"}
```

Comment:
comment attribution is fine, generally we like more medical sources than scientific american, but as this is about terminology in context of society perhaps it is ok